manifests/n/Nvidia/ProfileInspector/2.4.0.4/Nvidia.ProfileInspector.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Nvidia.ProfileInspector

```

```suggestion
PackageIdentifier: Orbmu2k.NvidiaProfileInspector
```
Best practice for package identifiers is `Publisher.PackageName` to avoid confusion. After updating this, you will need to also update the folder and name of the manifest files

---

Thanks, I'll keep that in mind and follow it up in my future PRs.
My hands are too full right now, but I'll get back to it when I'm a little freer.